shared/teams/common/selection-popup.tsx

```diff
@@ -265,6 +266,61 @@ const TeamMembersActions = ({teamID}: TeamActionsProps) => {
   )
 }
 
+const emptySetForUseSelector = new Set<string>()
+function allSameOrNull<T>(arr: T[]): T | null {
+  if (arr.length === 0) {
+    return null
+  }
+  const first = arr[0]
+  return arr.some(r => r !== first) ? null : first
+}
+const EditRoleButton = ({members, teamID}: {teamID: Types.TeamID; members: string[]}) => {
+  const dispatch = Container.useDispatch()
+
+  const teamDetails = Container.useSelector(state => Constants.getTeamDetails(state, teamID))
+  const roles = members.map(username => teamDetails.members.get(username)?.type)
+  const currentRole = allSameOrNull(roles)
```

Comment:
Makes sense for it to do this. I think we need to `setRole` to this when we show the popup or it's only set on mount.